### DISCUSSION TITLE
Remove Publish and Ratify proposal

### DISCUSSION THREAD

**Discussion by plor** [odyssy-automaton/pokemol-web#1895] [General]
Probably not the biggest decision to make, but the proposal now is to remove from the Poster Boost the option to post and ratify in the same proposal. This will require the user to post to IPFS first, and then create a proposal to ratify. Feedback about how valuable this option is and whether it is worth dev time to keep it around would be appreciated.

**Comment by earth2travis** [comment]
Seems like it would be nice to have. But combining the transactions is just a way to improve the flow. As long as we have the process well documented that walks users through the steps it should be fine for now. Let's revisit this once we get some usage and determine if the optimization is worth the effort.

**Comment by arentweall** [comment]
Agreed with the sentiments here. Let's revisit this later on, since the effort is considerable.
